### ISSUE TITLE
How to pass Mailgun options?

### ISSUE THREAD

**Issue by jzellis** [gfk-ba/meteor-mailgun-api#4]
I'm trying to schedule an email for future delivery. I tried doing something like this:
    `emailObj = {
    to: 'recipient@mail.com',
    'subject' : "This is an email",
    options: {deliverytime: theFutureDate
    }`

Etc. But the email is sent instantly. I also tried `options: {"o.deliverydate": theFutureDate}` and `Mailgun.send{emailObj, {deliverytime: theFutureDate})` and neither of those worked either.

What am I missing?

**Comment by marcodejongh** [comment]
Try:
```
emailObj = { to: 'recipient@mail.com', 'subject' : "This is an email",'o:deliverytime': theFutureDate }
```

The package doesn't sanitise the emailObj before it gets passed on to mailgun. 

It only adds/changes some stuff on the emailObject based on the options argument. But I only wrote code for the options I needed myself. So feel free to create a PR adding deliverytime to the options object which sets it as ```o:deliverytime``` on the emailObject.

I set it up this way to decouple my email interface from mailgun but also because I felt a lot of the options that where used by mailgun where hard to remember. Especially the yes/no booleans where very annoying.
